sdk/storage/azure-storage-file-share/tests/test_file_async.py

```diff
@@ -1597,6 +1597,36 @@ async def test_update_range_from_file_url_with_oauth(self, **kwargs):
         await destination_file_client.upload_range_from_url(
             source_blob_client.url, offset=0, length=512, source_offset=0, source_authorization=token)
 
+    @FileSharePreparer()
+    @recorded_by_proxy_async
+    async def test_update_range_from_file_url_copy_source_error_and_status_code(self, **kwargs):
```

Comment:
Should also add a sync equivalent test for this test. You can add it to `test_file.py`